wolfcrypt/benchmark/benchmark.c

```diff
@@ -110,7 +110,7 @@
     #undef  NO_FILESYSTEM
     #define NO_FILESYSTEM
 
-#elif defined(ANDROID)
+#elif defined(ANDROID) && !defined(ANDROID_v454)
```

Comment:
For consistency, I feel like we should make "v" uppercase (`ANDROID_V454`), as most of our defines are all uppercase.

Also, can we add a one-line comment above here mentioning why `ANDROID_V454` doesn't use our normal `ANDROID` logging here?

Reply:
Done.